docs/nginx-reverse-proxy.md

```diff
@@ -0,0 +1,99 @@
+# Nginx Reverse Proxy
+
+As already mentioned, this project relies on [pluswerk/docker-global](https://github.com/pluswerk/docker-global).
+
+Here is an information about the VIRTUAL_* variable clarified.
+
+## VIRTUAL_HOST: Virtual host (your domain)
+
+So that the nginx reverse proxy knows which domain belongs to which Docker container there is the VIRTUAL_HOST variable.
+
+##### Variant A
+
+You can simply specify this domain directly without regular expressions. See variant A.
+
+##### Regular expression information
+
+However, I would recommend a bit of regular expressions.
+
+The regular expression starts with a ~ and ends with a $$. So do not wonder why it's two dollars.
+
+If you have a DNS in between, you have the option to use multiple domains.
+For example, domain.vm & domain.vmd with different IP addresses.
+One goes to the local host, the other into a virtual machine. The possibilities are endless.
+
+##### Variant B
+
+For example, if you have a TYPO3 website, you could use variant B.
+With this you can use all subdomains if you have several websites in the project.
+
+##### Variant C + D
+
+But what if you have two Docker containers with the same domain?
+
+For example, you have a Symfony website domain.vm, but also a phpbb forum on forum.domain.vm.
+This means you create two directories with a docker-compose.yml for each project.
+
+You use variant C for the Symfony website, because domain.vm and www.domain.vm should be used.
+
+For the forum you use variant D, so you have forum.domain.vm.
+
+Finally, I can say that variant C or D is the better choice in most cases.
+But since every system is different, you can of course do what suits you.
+
+File docker-compose.yml:
+
+```yaml
```

Comment:
if the example file is on top I don't have to scroll so often.

Reply:
i think so too, the examples should go directly under the variants

Reply:
Done